pkg/feature/domain/feature.go

```diff
@@ -915,3 +919,44 @@ func updateStrategyVariationID(varID, uID string, s *feature.Strategy) error {
 	}
 	return nil
 }
+
+func (f *Feature) UpdateName(name string) error {
```

Comment:
If you are going to use [commands](https://github.com/bucketeer-io/bucketeer/pull/1036/files#diff-2d0a2c8153246509abb80eca89b6ca25f95f7ea8e75640dae22408489b8b0eafR717), why not use the current functions instead of creating new one doing the same thing?